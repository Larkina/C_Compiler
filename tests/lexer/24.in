1.444
10.5e+3
.e-2