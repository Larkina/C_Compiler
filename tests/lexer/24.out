value=1.444, text=1.444, type=FLOAT
value=10500.0, text=10.5e+3, type=FLOAT
value=0.0, text=0.e-2, type=FLOAT
